scripts/MAC Season 5.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "MAC Season 5 Metadata Upload to bety"
+output: html_document
+---
+
+
+########### Connect to bety database

```

Comment:
Use fewer `#`'s - appropriate for heading level, e.g. (from https://guides.github.com/features/mastering-markdown/)

```
# This is an <h1> tag
## This is an <h2> tag
###### This is an <h6> tag
```